public/codenotes.php

```diff
@@ -181,6 +182,7 @@ function saveCodeNote(rowIndex) {
         at address $00000000, immediately followed by the cartridge memory. As such, the addresses
         displayed below may not directly correspond to the addresses on the real hardware.</p>
         <br/>
+        <p>There are currently <b><?= $codeNoteCount?></b> code notes for this game.
```

Comment:
```suggestion
        <p>There are currently <span class='font-bold'><?= $codeNoteCount?></span> code notes for this game.</p>
```

nit: Tailwind instead of `<b>` tags. It also doesn't seem that this `<p>` tag has a closing `</p>`.

This page is probably a good candidate to migrate to Blade at the same time delete functionality is added to the page.